src/main/java/core/basesyntax/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.model.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Random RANDOM = new Random();
```

Comment:
```suggestion
    private final Random random = new Random();
```
don't need to make such objects static